rotkehlchen/api/rest.py

```diff
@@ -3931,6 +3931,8 @@ def add_manual_latest_price(
             )
 
         Inquirer().remove_cached_current_price_entry(cache_key=(from_asset, to_asset))
+        # also invalidate the usd price equivalent of the `from_asset`
+        Inquirer().remove_cached_current_price_entry(cache_key=(from_asset, A_USD))
```

Comment:
Wouldn't you need something similar to `pairs_to_invalidate` in delete? Since is not only USD but all the pairs with the from asset that need to be updated right?

Reply:
I don't think so. Or I don't understand you

Reply:
I mean that all the prices that depend on the from_asset should be removed from cache right?

Reply:
okay